basic_module/staking/src/execute.rs

```diff
@@ -276,3 +308,105 @@ pub fn change_params(
     metadata.save();
     Ok(outcome)
 }
+
+fn update_validators(validators: NextValidators) -> Result<TransactionExecutionOutcome, Error> {
+    let outcome: TransactionExecutionOutcome = Default::default();
```

Comment:
I cannot find a reason to declare this variable here.

Reply:
The original intention was add events to the outcome as the function proceeds. However, I'm not sure about what events to emit so now empty. I'll move the default() to the return posititon.